src/components/presentation/FeaturedArticles.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import FeaturedArticles from './FeaturedArticles';
+import renderWithRouter from '../../__mocks__/helpers';
+
+describe('<FeaturedArticles />', () => {
+  const articles = [{
+    id: 1,
+    title: 'something funny',
+    slug: 'something-funny',
+    author: { username: 'johndoe' },
+    imgUrl: 'https://img.io',
+    tags: [{ name: '' }],
+  }, {
+    id: 2,
+    title: 'other article',
+    slug: 'other-article',
+    author: { username: 'janedoe' },
+    imgUrl: 'https://img.io',
+    tags: [{ name: '' }],
+  },
+  ];
```

Comment:
Huh, If you had abstracted the articles above.
An import here would have fulfilled this instead of adding this additional const here.
Please modify.